packages/stems/src/assets/icons/iconLibrary.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M3.25 2.9375C3.25 3.45703 3.66797 3.875 4.1875 3.875H19.8125C20.332 3.875 20.75 3.45703 20.75 2.9375C20.75 2.41797 20.332 2 19.8125 2H4.1875C3.66797 2 3.25 2.41797 3.25 2.9375ZM3.875 8.25C3.33594 8.25 2.82422 8.48047 2.46875 8.88281C2.11329 9.28516 1.94922 9.82422 2.01563 10.3594L3.26563 20.3594C3.38282 21.2969 4.17969 22 5.125 22H18.875C19.8203 22 20.6172 21.2969 20.7344 20.3594L21.9844 10.3594C22.0508 9.82422 21.8867 9.28906 21.5313 8.88672C21.1758 8.48438 20.6641 8.25391 20.125 8.25391H3.875V8.25ZM19.1875 15.4375C19.1875 18.3711 15.9688 20.75 12 20.75C8.03125 20.75 4.8125 18.3711 4.8125 15.4375C4.8125 12.5039 8.03125 10.125 12 10.125C15.9688 10.125 19.1875 12.5039 19.1875 15.4375ZM12 16.6875C12.6914 16.6875 13.25 16.2695 13.25 15.75C13.25 15.2305 12.6914 14.8125 12 14.8125C11.3086 14.8125 10.75 15.2305 10.75 15.75C10.75 16.2695 11.3086 16.6875 12 16.6875ZM3.5625 5.125C3.04297 5.125 2.625 5.54297 2.625 6.0625C2.625 6.58203 3.04297 7 3.5625 7H20.4375C20.957 7 21.375 6.58203 21.375 6.0625C21.375 5.54297 20.957 5.125 20.4375 5.125H3.5625Z" fill="currentColor"/>
```

Comment:
Noting the use of `currentColor` here. I just did that in another branch, but it breaks from the established pattern on `web` of setting `fill` on paths through CSS modules (requires using the `color` property instead).

Personally, I prefer this pattern and am all about it. But it also differs from mobile, where the icons use `#000` and `#f00` as placeholders which are replaced via React props using SVGR. I think there's a chance we _might_ standardize on that in the future.

cc @dylanjeffers